src/movie.c

```diff
@@ -2630,20 +2678,31 @@ EXTERN_MSC int GMT_movie (void *V_API, int mode, void *args) {
 		}
 		if (error) {
 			GMT_Report (API, GMT_MSG_ERROR, "Running cleanup script %s returned error %d - exiting.\n", cleanup_file, error);
-			Return (GMT_RUNTIME_ERROR);
+			error = GMT_RUNTIME_ERROR;
+			goto out_of_here;
 		}
 	}
 
 	if (Ctrl->Z.delete) {	/* Delete input scripts */
 		if ((error = movie_delete_scripts (GMT, Ctrl)))
-			Return (error);
+			goto out_of_here;
 	}
 
 	/* Finally, delete the clean-up script separately since under DOS we got complaints when we had it delete itself (which works under *nix) */
 	if (!Ctrl->Q.active && gmt_remove_file (GMT, cleanup_file)) {	/* Delete the cleanup script itself */
 		GMT_Report (API, GMT_MSG_ERROR, "Unable to delete the cleanup script %s.\n", cleanup_file);
-		Return (GMT_RUNTIME_ERROR);
+		error = GMT_RUNTIME_ERROR;
+		goto out_of_here;
 	}
 
-	Return (GMT_NOERROR);
+out_of_here:
+	/* Remove the environmental reminder we are building a movie */
+#ifdef WIN32
```

Comment:
I never know but think it's safer to use _WIN32 then WIN32. Latter may also be used by Cygwin.

Reply:
Well, I copy/pasted this from somewhere else so if you would like to go through all of those and advice on which ones should be _WIN32 then be my guest.  I think in 2022 the burden lies on those 7 Cygwin uses to show problems.  As long as it is Windows compliant I think we are OK for now.

Reply:
The shit is that I don't know. I know that ``_WIN32`` is VisualStudio for sure and that MinGW also defines it. Also seem to understand that MinGW is a mix of VS and GCC libs ... and here starts the *bordel*

Reply:
It is pretty harmless.  I realize the reason I got that error was that I used the wrong -R in a place so what I was plotting was indeed exiting the frame. But even so we never want the movie frame to rotate...